keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,180 @@
+
+#### Story 3 (actual)
+
+Rafael wanted to share a Kubernetes course from an online education provider. He submitted [PR #15962](https://github.com/kubernetes/website/pull/15962) to add the course to [Overview of Kubernetes Online Training](https://kubernetes.io/docs/tutorials/online-training/overview/). The PR was not approved because SIG Docs didn’t want to add a link to third-party content over which SIG Docs have no control.
+
+#### Story 4 (actual)
+
+Website [PR #16203](https://github.com/kubernetes/website/pull/16203) removes Stackdriver and Elasticsearch vendor content. Since logging falls into the external add-ons category, SIG Docs decided to remove this vendor-specific content that had not been meaningfully updated in three years. SIG Docs had buy-in from SIG Instrumentation Bugs for removal; however that PR was held pending the outcome of this KEP.
+
+#### Story 5 (actual)
+
+In [PR #16766](https://github.com/kubernetes/website/pull/16766) @pouledodue proposed adding Hertzner Cloud Controller to the list of vendors that have implemented a cloud controller manager. That PR was held pending the outcome of this KEP.
+
+#### Story 6 (actual)
+
+As [hyperkube transitions to [third-party maintenance](https://github.com/kubernetes/kubeadm/issues/1889), it's unclear how to handle [hyperkube content in the Kubernetes docs](https://github.com/kubernetes/website/search?q=hyperkube&unscoped_q=hyperkube) or re-point related links.
+
+### Implementation Details/Notes/Constraints
+
+While SIG Docs approvers may occasionally need to consider intent, any policies produced by this KEP must be clear enough to minimize cases where intent needs to be considered.
+
+(At the time of writing) Kubernetes requires external software to implement Cluster Networking, Ingress, Persistent Storage, and Logging. Hyperlinking to vendor software and documentation _is_ allowed; creating “how to use” content is not.
+
+Examples of allowed content:
+ - Cluster Networking
+   - https://kubernetes.io/docs/concepts/cluster-administration/networking/
+   - https://kubernetes.io/docs/concepts/cluster-administration/addons/
+ - Ingress Controllers
+   - https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/#additional-controllers
+ - Persistent Volumes
+   - https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims
+
+Examples of content that would not be allowed:
+ - [Install a Network Policy Provider](https://kubernetes.io/docs/tasks/administer-cluster/network-policy-provider/) and child pages: how to use Calico, Cilium, Kube-router, Romana, and Weave Net for NetworkPolicy
```

Comment:
Network policy is an interesting one. I can understand pushing back on this particular doc which describes a specific third-party implementation, but would we keep a doc like this page with it's list of supported providers? https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy/

I think there's some grey area here.

Reply:
Agree - there is no "built-in" implementation.  But there's an implication that things listed here are "real" or "supported by the project", which is not necessarily true.  It also becomes an objective for new implementation - get listed or remain unknown.